src/gatsby-node.js

```diff
@@ -17,16 +17,27 @@ const headersWithToken = (headers, token) => {
   return headers;
 };
 
+// Helper to add expansions parameters
+const urlWithExpansions = (url, expansions) => {
+  if (expansions) {
+    return `${url}?expand=${expansions.join()}`;
+  }
```

Comment:
This is opinionated, but this kind of is statements might be easier to read with explicit else

```
} else { 
     return ...;
}
```

The shortest version would be using ternary operator `return foo ? condition : bar;`. Which one is better (more readable) depends on context.

Reply:
Yes, that's quite confusing actually. I've been following the Airbnb guide for a quite a while and that's what I've used here. 

What do you think would be the best to go with? I think since the ternary operator one is quite self-explanatory and concise, we could go with that?